core/iwasm/common/wasm_runtime_common.h

```diff
@@ -1210,6 +1210,12 @@ void
 wasm_runtime_set_linux_perf(bool flag);
 #endif
 
+bool
+is_valid_value_type(uint8 value_tpye);
+
+bool
+is_valid_func_type(const WASMFuncType *func_type);
```

Comment:
Just found that there are `wasm_loader_common.h` and `wasm_loader_common.c` under core/iwasm/common, do you think it is better to put these two functions there?

Reply:
looks better.